lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -35,8 +35,9 @@ module FOSSUtils
         #                            or a role (String or Symbol) that identifies one or more hosts.
         def configure_foss_defaults_on( hosts )
           block_on hosts do |host|
-            if (host[:version] && (not version_is_less(host[:version], '4.0'))) or host['type'] && host['type'] =~ /aio/
-              # add foss defaults to host
+            if (agent_only(host) && host[:version] && (not version_is_less(host[:version], '4.0'))) \
+              or (host['type'] && host['type'] =~ /aio/)
```

Comment:
I'm doing a little future proofing for the potential creation of pe-aio and foss-aio (aio agent for pe master, aio agent for foss master).